README.md

```diff
@@ -5,12 +5,21 @@ over the web.
 
 ## Concept
 
-The key idea is to have the respective distro's image copied to RAM and preserve
-it throughout a kexec. That is achieved by using a pmem device, which is created
+The webboot bootloader works as follows:
+
+1. fetch an OS distro release ISO from the web
+2. copy the ISO to memory
```

Comment:
memory -> persistent memory, e.g. /dev/pmem0

Reply:
I wanted to keep that detail for the full explanation below, so it doesn't confuse people early who don't know that concept